ch_backup/logic/access.py

```diff
@@ -48,11 +79,35 @@ def restore(self, context: BackupContext) -> None:
             return
 
         has_replicated_access = self._has_replicated_access(context)
-        mark_to_rebuild = not has_replicated_access
 
-        self._restore_local(acl_ids, context, mark_to_rebuild=mark_to_rebuild)
-        if has_replicated_access:
-            self._restore_replicated(acl_ids, acl_meta, context)
+        clickhouse_access_path = context.ch_ctl_conf["access_control_path"]
+        restore_tmp_path = os.path.join(
+            context.ch_ctl_conf["tmp_path"], context.backup_meta.name
+        )
+        user = context.ch_ctl_conf["user"]
+        group = context.ch_ctl_conf["group"]
+
+        create_dir(clickhouse_access_path, user, group)
+        with temporary_directory(restore_tmp_path, user, group):
+            if (
```

Comment:
Is this code needed in case of `has_replicated_access=True` ?

Reply:
Which one? This?
https://github.com/yandex/ch-backup/blob/5dfdf661279defd79cbe35536961dee661792ed1/ch_backup/logic/access.py#L92-L95

Or
https://github.com/yandex/ch-backup/blob/5dfdf661279defd79cbe35536961dee661792ed1/ch_backup/logic/access.py#L90

Reply:
https://github.com/yandex/ch-backup/blob/5dfdf661279defd79cbe35536961dee661792ed1/ch_backup/logic/access.py#L92-L95